cpp/src/centrality/betweenness_centrality_impl.cuh

```diff
@@ -266,6 +288,144 @@ void accumulate_vertex_results(
   }
 }
 
+template <typename vertex_t, typename edge_t, typename weight_t, bool multi_gpu>
+void accumulate_edge_results(
+  raft::handle_t const& handle,
+  graph_view_t<vertex_t, edge_t, false, multi_gpu> const& graph_view,
+  std::optional<edge_property_view_t<edge_t, weight_t const*>> edge_weight_view,
+  edge_property_t<graph_view_t<vertex_t, edge_t, false, multi_gpu>, weight_t>& centralities,
```

Comment:
If you're just changing the array contents, better create a mutable_view and pass the mutable view (https://github.com/rapidsai/cugraph/blob/branch-23.08/cpp/include/cugraph/edge_property.hpp#L134)

Reply:
Fixed in next push.